manage-data/lifecycle/index-lifecycle-management/migrate-index-management.md

```diff
@@ -27,12 +31,25 @@ To configure ILM Migration in the console:
 3. Near the top of the deployment overview, you should get a message to migrate from index curation to index lifecycle management (ILM) along with a **Start migration** button.
 4. Select which index curation pattern you wish to migrate.
 5. Set the ILM policy name for each data view (formerly *index pattern*).
+:::
+
+:::{tab-item} Elastic Cloud Enterprise
+1. [Log into the Cloud UI](../../../deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md).
+2. From the **Deployments** page, select your deployment.
+
+    Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
```

Comment:
Just a tiny nitpick. I can't add a comment at line 29, but for consistency I would replace it with this text here at line 40.

I'm pretty sure that the "Deployments" page looks the same in both ECH and Elastic Cloud Enterprise, so the text might as well be the same.